src/lib/GraphQL/Schema/InputHandler.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\EzPlatformMatrixFieldtype\GraphQL\Schema;
+
+use eZ\Publish\SPI\FieldType\Value;
+use EzSystems\EzPlatformGraphQL\GraphQL\Mutation\InputHandler\FieldTypeInputHandler;
+use EzSystems\EzPlatformMatrixFieldtype\FieldType\Value as MatrixValue;
+
+class InputHandler implements FieldTypeInputHandler
+{
+    public function toFieldValue($input, $inputFormat = null): Value
+    {
+        return new MatrixValue($input);
+    }
+}
```

Comment:
Don't we have bots for this ? :D

Reply:
> Don't we have bots for this ? :D

We do, but not here. Travis is not configured for this repo. Keep in mind though that it will work for .php files only and I already see missing line feed in some yaml also in this PR.
It's best that you enable adding new line in PhpStorm (the setting is called "ensure line feed at file end on Save").

Reply:
Enabled, thank you very much.